pkg/apps/blackduck/latest/containers/imageTags.go

```diff
@@ -48,6 +48,11 @@ func (c *Creater) GetImageTag(name string) string {
 	if len(confImageTag) > 0 {
 		return confImageTag
 	}
+
+	if len(c.hubSpec.RegistryConfiguration.Registry) > 0 && len(c.hubSpec.RegistryConfiguration.Namespace) > 0 {
```

Comment:
Similar to the below if statement, do we need to check whether the name is present in the imageTags?